coordinator/src/services/communication/tracker/translation.ts

```diff
@@ -197,11 +201,12 @@ function convertLineItemToInput(lineItemId: string, prescription: DetailPrescrip
 }
 
 function convertLineItemToOutput(lineItemId: string, prescription: DetailPrescription) {
-  const lineItem = prescription.lineItems[lineItemId]
   const taskOutput: fhir.TaskOutput = {
-    type: fhir.createCodeableConcept("http://snomed.info/sct", lineItem.code, lineItem.description),
+    type: fhir.createCodeableConcept("http://snomed.info/sct", "373784005", "Dispensing medication"),
     valueReference: fhir.createIdentifierReference(
-      fhir.createIdentifier("https://fhir.nhs.uk/Id/prescription-dispense-item-number", lineItemId.toLowerCase())
+      fhir.createIdentifier("https://fhir.nhs.uk/Id/prescription-dispense-item-number", lineItemId.toLowerCase()),
```

Comment:
Is the line item ID both the prescription-order-item-number AND the prescription-dispense-item-number?